client/templates/course/toolbar/edit-course.html

```diff
@@ -1,10 +1,15 @@
 <template name="editCourse">
     <ul class="nav nav-pills navbar-right" role="tablist">
         <li>
+        	{{#if owner}}
```

Comment:
Consider renaming this helper to 'canEditCourse'. That way the markup reads 'if can edit course', which allows for our model of collaborative editing as opposed to ownership.